src/ensembl/src/shared/state/ens-object/ensObjectHelpers.ts

```diff
@@ -109,5 +121,69 @@ export const buildRegionObject = (
   };
 };
 
+export const buildEnsObject = (
```

Comment:
Currently, ensObjects can be either of type gene or of type region. This function builds an object of a gene type. It looks like it deserves a better name than this.

Reply:
Renamed this to `buildGeneObject` ✅